resources/views/components/alert.blade.php

```diff
@@ -0,0 +1,37 @@
+<div {{ $attributes->class($alertClasses) }}>
+    <div class="flex">
+        @if ($icon)
+            <div class="flex-shrink-0">
+                <x-dynamic-component
+                    :component="WireUi::component('icon')"
+                    :name="$icon"
+                    class="h-5 w-5 shrink-0 {{ $subjectColor }}"
+                />
+            </div>
+        @endif
+
+        <div class="ml-3">
+            @if ($title)
+                <h3 class="mb-2 text-sm font-semibold {{ $subjectColor }}">
+                    {{ $title }}
+                </h3>
+            @endif
+
+            <div class="text-sm {{ $subjectColor }}">
+                <p>{{ $message ?? $slot }}</p>
```

Comment:
why it's not `{!! $message ?? $slot !}}` ? in this case we could add a link/span/etc. in message, now it doesn't render html tags :)